packages/kolibri-tools/lib/cli.js

```diff
@@ -331,4 +330,28 @@ program
     require('jest-cli/build/cli').run();
   });
 
-program.parse(process.argv);
+// Check engines, then process args
+const engines = require(path.resolve(__dirname, '../../../package.json')).engines;
+checkVersion(engines, (err, results) => {
+  if (err) {
+    console.error(err);
```

Comment:
Oh, and use the logger module imported above, that will do standard colour coding for errors for you, and will also mean you don't get `no-console` linting breakages.

`cliLogging.error` will apply the right colouring without needing the `colors` require at the top.

Reply:
If it's okay with you, I would like to align the logger more closely to how the `console` works. Some proposed changes:
- Allow logger methods to accept multiple messages as different arguments, as does `console`
- Add `.format()` method which will help facilitate color formatting and prefixing
- Use corresponding `console` methods for each, so that messages are output to stdout and stderr respectively; `.error()` and `.warn()` should go to stderr, and the rest should go to stdout.

Reply:
Yep - fine by me! You may be the first person apart from me to ever look at the file, so it has not received much TLC.